src/main/java/ca/corefacility/bioinformatics/irida/model/sequenceFile/Fast5Object.java

```diff
@@ -92,16 +93,17 @@ public void setFast5Type(Fast5Type fast5Type) {
 	/**
 	 * Get the {@link Fast5Type} for this object
 	 *
-	 * @param isGzipped if the file type is gzipped or not
+	 * @param file if the file
+	 * @return type of fast5object (unknown, zipped, or single)
 	 */
-	public void setType(boolean isGzipped) {
+	public Fast5Type setType(SequenceFile file) {

```

Comment:
This should likely be a `private` method given the way it's being used.

Reply:
Updated in 97e0014e1c5aa5584ad4b4f07bf67f3d54fbaf70